earthpy/clip.py

```diff
@@ -72,8 +69,7 @@ def _clip_line_poly(shp, clip_obj):
 
 
 def clip_shp(shp, clip_obj):
-    """A function to clip points, lines, polygon geometries based on an input
-    geometry.
+    """Clip points, lines, or polygon geometries.
```

Comment:
How about: 
`Clip points, lines, or polygon geometries to the clip_obj geometry.`